README.md

```diff
@@ -1035,6 +1059,20 @@ This example playbooks sets the `firewall` variables for the `fedora.linux_syste
     - microsoft.sql.server
 ```
 
+### Setting Up SQL Server with TLS Encryption using self signed certificate and key created by the certificate role
+
+    - name: Test with certs created by the certificate role
+      include_role:
+        name: linux-system-roles.mssql
+        public: true
```

Comment:
Is setting `public: true` strictly required?

Reply:
probably not - I'm assuming this is a copy/paste from a test